model_compression_toolkit/core/common/mixed_precision/search_methods/linear_programming.py

```diff
@@ -64,7 +66,10 @@ def mp_integer_programming_search(search_manager: MixedPrecisionSearchManager,
                                     target_kpi,
                                     search_manager)
 
-    lp_problem.solve()  # Try to solve the problem.
+    # Use default PULP solver. Limit runtime in seconds
+    solver = PULP_CBC_CMD(timeLimit=SOLVER_TIME_LIMIT)
```

Comment:
Are you sure that this is the solver that has been running so far? wasn't it running multiple algorithms in order to get the best solution? (I'm just verifying that this wouldn't change the behavior)

Reply:
Yeah, I'm sure. PuLP default solver selection process is:

# Default solver selection
if PULP_CBC_CMD().available():
    LpSolverDefault = PULP_CBC_CMD()
elif GLPK_CMD().available():
    LpSolverDefault = GLPK_CMD()
elif COIN_CMD().available():
    LpSolverDefault = COIN_CMD()
else:
    LpSolverDefault = None